misaligned.cpp

```diff
@@ -1,13 +1,29 @@
 #include <iostream>
 #include <assert.h>
+#include <iomanip>
+
+#include "misaligned.hpp"
 
 int printColorMap() {
     const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
     const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
+
+    int numberOfMinorColors = sizeof(minorColor) / sizeof(minorColor[0]);
+    
     int i = 0, j = 0;
     for(i = 0; i < 5; i++) {
         for(j = 0; j < 5; j++) {
-            std::cout << i * 5 + j << " | " << majorColor[i] << " | " << minorColor[i] << "\n";
+            std::cout << std::left << std::setw(MAX_WIDTH) <<i * 5 + j + 1 << " | " << std::setw(MAX_WIDTH)
+            << majorColor[i] << " | " << std::setw(MAX_WIDTH)<< minorColor[j] << "\n";
+
+            
+            int PairNum = ColorNum_To_PairNum(i, j, numberOfMinorColors);
+
+            assert(PairNum == i * 5 + j + 1 ); // add 1 to get the correct Pairno index

```

Comment:
The `assert` checks the result of the macro, while the production code isn't using the macro. Hence the test may pass, but the report can still be wrong